data/remnant/remnant 1 introduction.txt

```diff
@@ -1636,6 +1636,8 @@ mission "Remnant: Tech Retrieval"
 	on offer
 		"remnant met taely" ++
 		conversation
+			branch familiar
+				has "remnant met taely"
```

Comment:
Should this not be called `remnant: met taely`?

Reply:
This is the name of the condition that is currently used. Perhaps it should be renamed to that, but I don't think that's a question this PR needs to answer.

Reply:
Since the condition is incremented directly before the conversation, is it not safe to assume that the player has it? If not, the incrementing should be done after the branch check.

Reply:
The content of a mission action is not necessarily executed in the order it is listed.
The relevant part of this here is that a conversation panel is constructed before any condition changes in the mission action are applied. During construction, the conversation panel finds the first text it should display, which involves evaluating and following any branches it finds prior to visible text.
Here, that means this branch will check the condition before the increment operation is applied.